ibikecph-lib/src/main/java/com/spoiledmilk/ibikecph/login/LoginSplashActivity.java

```diff
@@ -114,78 +114,78 @@ public void launchMainMapActivity() {
+                            finish();
+
+                        }
+                    });
+                } else {
+                    LoginSplashActivity.this.runOnUiThread(new Runnable() {
+                        @Override
+                        public void run() {
+                            findViewById(R.id.progressBar).setVisibility(View.GONE);
+                            launchMainMapActivity();
+                        }
+                    });
+                }
+            }
+        }).start();
     }
 
     public void onBtnLogInClick(View v) {
-//        Intent i = new Intent(this, LoginActivity.class);
-//        startActivityForResult(i, LOGIN_REQUEST);
+        Intent i = new Intent(this, LoginActivity.class);
+        startActivityForResult(i, LOGIN_REQUEST);
     }
 
     public void onEnableTrackingClick(View v) {
 
-//        if (IBikeApplication.getSignature().equals("")) {
-//            if (IBikeApplication.isFacebookLogin()) {
-//                Log.d("DV", "Prompting Facebookuser to create a password!");
-//                Intent i = new Intent(LoginSplashActivity.this, SignatureActivity.class).addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
-//                i.putExtra("fromSplashScreen", true);
-//                IBikeApplication.setWelcomeScreenSeen(true);
-//                startActivity(i);
-//            } else if (IBikeApplication.isUserLogedIn()) {
-//                Log.d("DV", "Prompting login for user!");
-//                Intent i = new Intent(LoginSplashActivity.this, SignatureActivity.class).putExtra("normalUser", true).addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
-//                IBikeApplication.setWelcomeScreenSeen(true);
-//                startActivity(i);
-//            }
-//        } else {
-//            Log.d("DV", "We got a signature, enabling tracking!");
-//            IBikeApplication.getSettings().setTrackingEnabled(true);
-//            launchMainMapActivity();
-//        }
+        if (IBikeApplication.getSignature().equals("")) {
+            if (IBikeApplication.isFacebookLogin()) {
+                Log.d("DV", "Prompting Facebookuser to create a password!");
+                Intent i = new Intent(LoginSplashActivity.this, SignatureActivity.class).addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
+                i.putExtra("fromSplashScreen", true);
+                IBikeApplication.setWelcomeScreenSeen(true);
+                startActivity(i);
+            } else if (IBikeApplication.isUserLogedIn()) {
+                Log.d("DV", "Prompting login for user!");
+                Intent i = new Intent(LoginSplashActivity.this, SignatureActivity.class).putExtra("normalUser", true).addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
+                IBikeApplication.setWelcomeScreenSeen(true);
+                startActivity(i);
+            }
+        } else {
+            Log.d("DV", "We got a signature, enabling tracking!");
+            IBikeApplication.getSettings().setTrackingEnabled(true);
```

Comment:
not clear to me when this happens, and kind of tracking is turned on?

Reply:
As commented on the other pull request, The tracking is disabled. It's part of the tracking project, that we discussed, we are not going to use.

Reply:
we'll clean it up later. as long as you're sure it's currently turned off. all this tracking should really have been done in a feature branch